Updating Live previewing a theme (#469)
bigcommerce/dev-docs: docs/stencil-docs/installing-stencil-cli/live-previewing-a-theme.md

</div>

> Once Stencil CLI is installed, the next step on the road to theme development is downloading a theme to edit and previewing live changes using Stencil CLI's powerful Browsersync functionality. This article will walk you through the process of downloading a theme for development, installing theme modules, then serving a live preview using Stencil CLI's `stencil start` command. The steps in this article assume Stencil CLI has been installed on your system; if it's not installed yet, see [Installing Stencil CLI](https://developer.bigcommerce.com/stencil-docs/installing-stencil-cli/installing-stencil) for detailed, operating system specific instructions.

Reviewer: ```suggestion
Once you've installed the Stencil CLI, the next steps are downloading a theme to edit and previewing live changes using Stencil CLI's powerful Browsersync functionality. This article walks you through the process of downloading a theme for development, installing theme modules, and serving a live preview using Stencil CLI's `stencil start` command. 

The steps in this article assume you've installed Stencil CLI on your system. If you haven't installed it yet, see [Installing Stencil CLI](https://developer.bigcommerce.com/stencil-docs/installing-stencil-cli/installing-stencil) for detailed, system-specific instructions.

```

## Obtaining Store API credentials

> Stencil CLI uses various BigCommerce APIs to inject store specific data (like carousel images and products) into the live theme preview it serves up. To do so, Stencil CLI must be supplied with a store API token. [Store API Accounts](https://support.bigcommerce.com/s/article/Store-API-Accounts) and their tokens are generated in **Advanced Settings** > **API Accounts** inside a BigCommerce store's control panel (see [Store API Accounts](https://support.bigcommerce.com/s/article/Store-API-Accounts) for detailed instructions). 

Reviewer: ```suggestion
Stencil CLI uses various BigCommerce APIs to inject store-specific data, like carousel images and products, into the live theme preview it serves up. To do so, you must supply the Stencil CLI with a store API token. For detailed instructions, see [Store API Accounts](https://support.bigcommerce.com/s/article/Store-API-Accounts). 

```

<!-- theme: warning -->

### Distribution of Cornerstone-Based Themes
> Distribution of Cornerstone-based themes is subject to BigCommerce's Cornerstone license, including the mandatory incorporation of BigCommerce's copyright statement.

### Back Up before Reinstalling

> > If you're re-installing an existing theme, be sure to backup the theme’s `.stencil` file -- this contains the store URL, username, access tokens, and other settings. If you would like to allow for complete rollback, back up your entire theme’s directory.

Reviewer: ```suggestion
> If you're re-installing an existing theme, be sure to back up the theme’s `.stencil` file. This file contains the store URL, username, access tokens, and other settings. If you would like to allow for complete rollback, back up your entire theme’s directory.

```

<!-- theme: info -->

> ### Themes Prior to 1.10.0

Reviewer 2: Can we also get rid of this page https://developer.bigcommerce.com/legacy/stencil-themes/installing-legacy-theme-modules

Author: Hi Juned, My understanding is that a lot of the Stencil documentation is old and really needs to be looked at and reviewed.  I would like to create a separate ticket to identify these items.  I would like to close this ticket and open up a new one listing all obsolete information that needs to be removed. I can add this page to it. Would that be okay?

Reviewer 2: Yes this is not a blocker for this one.

<!-- theme: info -->

### Themes Prior to 1.10.0

> > For instructions on installing modules via `jspm` on pre 1.10.0 themes, see [Installing Legacy Theme Modules](https://developer.bigcommerce.com/legacy/stencil-themes/installing-legacy-theme-modules).

Reviewer 2: @bookernath should we completely deprecate this section since it will be no longer valid for new theme development. Also the page it is linking to ?

Reviewer 3: I agree. It's been a few years, let's remove references to jspm.
